spec/README.md

```diff
@@ -7,7 +7,7 @@ Because we think that:
 
 - UNIX® is great and we love it
 - The UNIX® shell is great
-- HTTP interfaces are convenient and everywhere
+- HTTP interfaces are convenient and ubiquitous
```

Comment:
-0, not [Basic English](https://simple.wikipedia.org/wiki/Basic_English)

Reply:
Fair enough.  Arigato!

Reply:
(reverted)